code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java

```diff
@@ -75,7 +74,7 @@ public Account getAccountByExternalId(String externalId, String externalType) {
                 .where(
                         ACCOUNT.EXTERNAL_ID.eq(externalId)
                         .and(ACCOUNT.EXTERNAL_ID_TYPE.eq(externalType))
-                        .and(ACCOUNT.STATE.eq("active"))
+                        .and(ACCOUNT.STATE.ne("purged"))
```

Comment:
spoke to @vincent99 ... all projects not in "purged" state should be shown in the API response